course04/homework00/ciaranfrain/CMakeLists.txt

```diff
@@ -0,0 +1,4 @@
+cmake_minimum_required (VERSION 2.6)
+project (SortedArray)
+file(GLOB SortedArraySources "*.h" "*.cpp")
```

Comment:
It's not recommended to use GLOB. The official cmake doc doesnt recommend it, iirc mostly because it wont correctly detect when files are being added, but most importantly the globbing might add some files you wouldnt add explicitly to the project - in such simple usage it wont happen obviously, but in big projects it happened to me.